Add notebook of faling object example (#161)
CUQI-DTU/CUQI-Book: chapter00/section5.md

now dominated by the prior.
This is all good if the mean if well chosen - but it can also be
dangerous if we have too much confidence in a bad prior that does not

> reflect reality. A notebook that implements the falling object problem in CUQIpy can be found in the appendix at {ref}`falling_object_demo`.

Reviewer: ```suggestion
reflect reality. A full implementation of the falling object problem in CUQIpy can be found in the appendix at {ref}`falling_object_demo`.
```
